src/runtime/composables/local/useAuth.ts

```diff
@@ -109,11 +109,15 @@ const signUp = async (credentials: Credentials, signInOptions?: SecondarySignInO
   const nuxt = useNuxtApp()
 
   const { path, method } = useTypedBackendConfig(useRuntimeConfig(), 'local').endpoints.signUp
-  await _fetch(nuxt, path, {
+  const response = await _fetch<Record<string, any>>(nuxt, path, {
     method,
     body: credentials
   })
 
+  if(signInOptions?.stopLogin) {
+    return response
```

Comment:
I would not return different values based on if you want the signIn flow to happen or not, as I feel like this would lead to confusion.